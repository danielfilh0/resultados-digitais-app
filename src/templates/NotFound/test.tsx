import { screen } from '@testing-library/react'

import { NotFound } from '.'
import { renderWithTheme } from '@/utils/tests/helpers'

describe('<NotFound />', () => {
  it('should render the heading', () => {
    const { container } = renderWithTheme(<NotFound />)

    expect(
      screen.getByRole('heading', { name: /404 - página não encontrada/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
