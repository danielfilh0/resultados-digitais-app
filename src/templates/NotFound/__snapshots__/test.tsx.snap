// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should render the heading 1`] = `
.c2 {
  color: #FFFFFF;
  text-align: center;
  font-size: 2.4rem;
}

.c0 {
  padding: 4.8rem 0;
}

.c1 {
  max-width: 128rem;
  width: 100%;
  margin: 0 auto;
  padding: 0 2rem;
}

@media (min-width: 768px) {
  .c2 {
    font-size: 4rem;
  }
}

<section
  class="c0"
>
  <div
    class="c1"
  >
    <h1
      class="c2 nunito"
    >
      404 - Página não encontrada
    </h1>
  </div>
</section>
`;
